testing/trino-product-tests/src/main/java/io/trino/tests/kafka/TestKafkaAvroReadsSmokeTest.java

```diff
@@ -73,32 +72,9 @@
     private static final String STRUCTURAL_AVRO_TOPIC_NAME = "read_structural_datatype_avro";
     private static final String STRUCTURAL_SCHEMA_PATH = "/docker/presto-product-tests/conf/presto/etc/catalog/kafka/structural_datatype_avro_schema.avsc";
 
-    private static void createAvroTable(String schemaPath, String tableName, String topicName, ImmutableMap<String, Object> record, MessageSerializer messageSerializer)
-    {
-        try {
-            Schema schema = new Schema.Parser().parse(new File(schemaPath));
-            byte[] avroData = messageSerializer.serialize(topicName, schema, record);
-
-            KafkaTableDefinition tableDefinition = new KafkaTableDefinition(
-                    KAFKA_SCHEMA + "." + tableName,
-                    topicName,
-                    new ListKafkaDataSource(ImmutableList.of(
-                            new KafkaMessage(
-                                    contentsBuilder()
-                                            .appendBytes(avroData)
-                                            .build()))),
-                    1,
-                    1);
-            KafkaTableManager kafkaTableManager = (KafkaTableManager) testContext().getDependency(TableManager.class, "kafka");
-            kafkaTableManager.createImmutable(tableDefinition, tableHandle(tableName).inSchema(KAFKA_SCHEMA));
-        }
-        catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
-    }
-
     @Test(groups = {KAFKA, PROFILE_SPECIFIC_TESTS}, dataProvider = "catalogs")
     public void testSelectPrimitiveDataType(KafkaCatalog kafkaCatalog, MessageSerializer messageSerializer)
+            throws Exception
```

Comment:
What has changed? Is it the correct commit?

Reply:
Previously we would throw `UncheckedIOException` so the test method wasn't aware of it.